tests/test_opds.py

```diff
@@ -921,14 +921,16 @@ def test_page_feed(self):
         work1 = self._work(genre=Contemporary_Romance, with_open_access_download=True)
         work2 = self._work(genre=Contemporary_Romance, with_open_access_download=True)
 
-        self.add_to_materialized_view([work1, work2], True)
+        search_engine = MockExternalSearchIndex()
+        search_engine.bulk_update([work1, work2])
+
         facets = Facets.default(self._default_library)
         pagination = Pagination(size=1)
 
         def make_page(pagination):
             return AcquisitionFeed.page(
                 self._db, "test", self._url, lane, TestAnnotator,
-                pagination=pagination
+                pagination=pagination, search_engine=search_engine
```

Comment:
Changing these tests was surprisingly easy. Instead of adding the works to the materialized view, I add them to a mock search index. Everything else works as-is.